front/src/App.js

```diff
@@ -23,8 +24,8 @@ function App() {
           <Route exact path="/dashboard">
             <DashboardPage />
           </Route>
-          <Route exact path="/url/list">
-            <UrlList />
+          <Route path="/url/list">
```

Comment:
We need to add exact path to prevent interference with the /{short_url} path used to unshorten links

```suggestion
          <Route exact path="/url/list">
```